deployment/terraform/coordinator.go

```diff
@@ -76,9 +73,14 @@ func (t *Terraform) StartCoordinator() error {
 
 	mlog.Info("Uploading other load-test config files")
 
-	agentConfig, err := t.generateLoadtestAgentConfig(output)
+	var agentConfig *loadtest.Config
+	if len(output.Instances) > 0 {
+		agentConfig, err = t.generateLoadtestAgentConfig(output)
```

Comment:
Do we just want to pass a boolean to the method to just return after doing `loadtest.ReadConfig("")`? It also does the same thing.

Reply:
I think both approaches are valid but I feel that being explicit at the cost of some minor repetition makes the code easier to follow. We either generate the config or not. I considered both options and thought that adding more state to the helper function just to avoid a couple of lines was not worth it.